src/p2p/SyncV2/index.ts

```diff
@@ -169,6 +177,14 @@ function syncStandbyNodeList(activeNodes: P2P.SyncTypes.ActiveNode[]): ResultAsy
   })
 }
 
+function syncTxList(activeNodes: P2P.SyncTypes.ActiveNode[]): ResultAsync<{ hash: string; tx: P2P.ServiceQueueTypes.AddNetworkTx }[], Error> {
+  return robustQueryForTxListHash(activeNodes).andThen(({ value, winningNodes }) =>
+    getTxListFromNode(winningNodes[0], value.txListHash).andThen((txList) =>
+      verifyTxList(txList, value.txListHash).map(() => txList)
+    )
+  )
+}
```

Comment:
i think this need to be more robust. Right now, we get the robust txlisthash and query and check just one node for the its txlist and verify that it matches. However, if it doesn't we should retry a few more times. I think retrying 5 times or the length of winningNodes should be sufficient, whichever is smaller

Reply:
I say this because if we get unlucky and ask a node that is OOS and has a different txlist, syncing looks like its gonna fail